website/server.py

```
@@ -0,0 +1,147 @@
+        async with httpx.AsyncClient() as client:
+            async with client.stream(
+                "POST",
+                f"{FASTCHAT_API_BASE}/v1/chat/completions",
+                json=payload,
+                timeout=120,
+            ) as resp:
+                if resp.status_code >= 400:
+                    try:
+                        err_body = await resp.aread()
+                        err_text = err_body.decode("utf-8", errors="replace")[:2000]
+                    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)